specification/frontdoor/resource-manager/Microsoft.Network/stable/2021-06-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,981 @@
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete successful."
+          },
+          "202": {
+            "description": "Accepted. The request has been accepted for processing and the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "No Content. The request has been accepted but the policy was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Delete protection policy": {
+            "$ref": "./examples/WafPolicyDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/FrontDoorWebApplicationFirewallManagedRuleSets": {
+      "get": {
+        "tags": [
+          "WebApplicationFirewallManagedRuleSets"
+        ],
+        "description": "Lists all available managed rule sets.",
+        "operationId": "ManagedRuleSets_List",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of all available web application firewall managed rule sets.",
+            "schema": {
+              "$ref": "#/definitions/ManagedRuleSetDefinitionList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List Policies in a Resource Group": {
+            "$ref": "./examples/WafListManagedRuleSets.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "WebApplicationFirewallPolicy": {
+      "description": "Defines web application firewall policy.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties of the web application firewall policy.",
+          "$ref": "#/definitions/WebApplicationFirewallPolicyProperties"
+        },
+        "etag": {
+          "type": "string",
+          "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        },
+        "sku": {
+          "description": "The pricing tier of web application firewall policy. Defaults to Classic_AzureFrontDoor if not specified.",
+          "$ref": "#/definitions/Sku"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
```

Comment:
>Resource [](http://example.com/codeflow?start=47&length=8)

You have a sku property on this so it feels like you want this to be billable in which case this needs to be a tracked resource